client/src/components/Draftail/blocks/EmbedBlock.test.js

```diff
@@ -15,6 +15,8 @@ describe('EmbedBlock', () => {
                 url: 'http://www.example.com/',
                 title: 'Test title',
                 thumbnail: 'http://www.example.com/example.png',
+                authorName: 'Example author',
+                providerName: 'Example provider',
```

Comment:
Could you add a separate case for this test suite that specifically tests what this component renders when there is data, but not a thumbnail?